editor/src/messages/portfolio/portfolio_message_handler.rs

```diff
@@ -180,10 +180,10 @@ impl MessageHandler<PortfolioMessage, (&InputPreprocessorMessageHandler, &Prefer
 						};
 
 						let node = layer.to_node();
+						let previous_alias = active_document.network().nodes.get(&node).unwrap().alias.clone();
```

Comment:
Please avoid this unwrap, perhaps defaulting to an empty string.

Reply:
I made this change, but please look at the code to learn from the updated code for your future reference.